keyboards/darmoshark/k3/info.json

```
@@ -0,0 +1,183 @@
+{
+    "manufacturer": "Darmoshark",
+    "keyboard_name": "K3 QMK",
+    "maintainer": "Proceee",
+    "url": "",
+    "processor": "WB32FQ95",
+    "bootloader": "wb32-dfu",
+    "usb": {
+        "device_version": "1.0.0",
+        "vid": "0xC001",
+        "pid": "0x3667",
+        "suspend_wakeup_delay": 1000
+    },
+    "diode_direction": "ROW2COL",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+    "eeprom": {
+        "driver": "wear_leveling",
+        "wear_leveling": {
+            "driver": "spi_flash",
+            "backing_size": 4096
+        }
+    },
+    "matrix_pins": {
+        "rows": ["B13", "A1", "A2", "A3", "A4", "B7"],
+        "cols": ["B1", "C7", "C13", "B9"]
+    },
+    "indicators": {
+        "num_lock": "C5",
+        "on_state": 1
+    },
+    "ws2812": {
+        "pin": "A8"
+    },
+    "rgb_matrix": {
+        "driver": "ws2812",
+        "default":{
+          "val": 80
+        },
+        "sleep": true.
```

Sorry, fixes previous bad suggesion 
```suggestion
        "sleep": true,
```